<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_dialog</class>
 <widget class="QDialog" name="edit_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>275</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="edit_dialog_hours">
        <property name="text">
         <string>Hours</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edit_dialog_input_hours"/>
      </item>
      <item>
       <widget class="QLabel" name="edit_dialog_minutes">
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edit_dialog_input_minutes"/>
      </item>
      <item>
       <widget class="QLabel" name="edit_dialog_seconds">
        <property name="text">
         <string>Secs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edit_dialog_input_seconds"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="edit_dialog_ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="edit_dialog_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
